R/tags.R

```diff
@@ -1668,7 +1690,7 @@ NULL
 
 #' @rdname knitr_methods
 #' @export
-knit_print.shiny.tag <- function(x, ...) {
+knit_print.shiny.tag <- function(x, ..., inline = FALSE) {
```

Comment:
For context, see `?knit_print` -- this argument has existed for at least 8 years https://github.com/yihui/knitr/commit/532f68d3e0aef336e4b10c3199ddf3dc11252745